tasks/el8/el8_base_deps.yml

```diff
@@ -60,26 +63,29 @@
 # root permissions. Using become and the raw shell command invokes these in root's environment,
 # which will fail at best, or silently map the wrong command at worst.
 - name: "Configure alternatives: python"
-  shell: sh -c 'sudo alternatives --set python $(which python3)'
+  ansible.builtin.shell: sh -c 'sudo alternatives --set python /usr/bin/python3'
```

Comment:
$(which python3) is giving the wrong path for some reason in these containers...